bin/ParseDarwinSamplesheet.sh

```diff
@@ -243,6 +243,11 @@ else
 			continue
 		fi
 		host_prm=$(hostname -s)
+
```

Comment:
Now that the data transfers with rsync have been relocated from ```bin/ConcordanceCheck.sh``` to this script it would be nice to update the description of what this script does in the comments and commandline help; E.g. this:
`Scripts to make automatically a samplesheet for the concordance check between ngs and array data.`
is incomplete.